requirements.txt

```diff
@@ -1,6 +1,6 @@
-git+https://github.com/slaclab/pydm.git
-git+https://github.com/pcdshub/QDarkStyleSheet.git
```

Comment:
Should we nuke our pcdshub/QDarkStyleSheet repo?

Reply:
This is probably a good idea.  I will need to change where we build it from in `pcds-recipes` as well.